assets/css/free-trial-admin.css

```diff
@@ -11,3 +11,9 @@
 .wc-calypso-notice a {
 	color: inherit;
 }
+
+.woocommerce-task-list__setup
+	.woocommerce-experimental-list
+	.woocommerce-experimental-list__item.complete {
+	color: var( --wp-admin-theme-color );
+}
```

Comment:
Will the fix only apply to sites in the free trial with the CSS placed in this file? I think we'll need to ensure it continues to work when sites are upgraded to a plan.

Reply:
That's a good feedback!

Although I've opened a [PR](https://github.com/woocommerce/woocommerce/pull/37001) in the core to make this change permanent, we should still make the change in wc-calypso-brige. The core change won't be included soon enough for our launch.

I'll make a change to it.

Reply:
@adrianduffell Updated in https://github.com/Automattic/wc-calypso-bridge/pull/1006/commits/1baceb3d1f10b9cab8624840014c30ce4d882655

Since we don't have s shared CSS file between ECOM and trial, I've added the same style to ecommerce.css.

It's a duplicate, but we can remove the style from both ECOM and trial once the core change gets merged.